langchain/package.json

```diff
@@ -673,6 +673,7 @@
   "license": "MIT",
   "devDependencies": {
     "@aws-crypto/sha256-js": "^5.0.0",
+    "@aws-sdk/client-bedrock-runtime": "^3.422.0",
```

Comment:
Didn't know this was out - we should probably switch our other Bedrock integrations to use this right?

Reply:
Yes, definitely. This went GA last week so it's very fresh.

Reply:
I can take a look at updating the other integrations as well .

Reply:
Is this not yet done? If so, would you mind calling out those spots. Might be able to help with a PR too.